src/StickyParallaxHeader.tsx

```diff
@@ -66,44 +149,49 @@ class StickyParallaxHeader extends Component {
             ? this.tabsScrollPosition[currentPage]
             : scrollHeight;
         const scrollNode = getSafelyScrollNode(this.scroll);
-        scrollNode.scrollTo({ y: scrollTargetPosition, duration: 1000 });
+        scrollNode?.scrollTo({ y: scrollTargetPosition });
       }, 250);
     }
   }
 
   componentWillUnmount() {
+    // https://reactnative.dev/docs/animatedvaluexy#removealllisteners
+    // @ts-ignore
     this.scrollY.removeAllListeners();
     this.props.onRef?.(null);
   }
 
   spring = () => {
     const scrollNode = getSafelyScrollNode(this.scroll);
-    scrollNode.scrollTo({ x: 0, y: 40, animated: true });
+    scrollNode?.scrollTo({ x: 0, y: 40, animated: true });
 
     return setTimeout(() => {
       setTimeout(() => {
-        scrollNode.scrollTo({ x: 0, y: 25, animated: true });
+        scrollNode?.scrollTo({ x: 0, y: 25, animated: true });
       }, 200);
-      scrollNode.scrollTo({ x: 0, y: 0, animated: true });
+      scrollNode?.scrollTo({ x: 0, y: 0, animated: true });
     }, 300);
   };
 
-  onScrollEndSnapToEdge = (height) => {
+  onScrollEndSnapToEdge = (height: number) => {
     const { snapStartThreshold, snapStopThreshold, snapValue } = this.props;
     const scrollHeight = snapStopThreshold || height;
     const snap = snapValue || height;
     const { snapToEdge, refreshControl } = this.props;
 
     const scrollNode = getSafelyScrollNode(this.scroll);
+    // @ts-ignore
     const scrollValue = this.scrollY.__getValue();
+    // const scrollValue = this.scrollY;
     const { y } = scrollValue;
     const snapToEdgeAnimatedValue = new ValueXY(scrollValue);
     const snapToEdgeThreshold = snapStartThreshold || height / 2;
     const id = snapToEdgeAnimatedValue.addListener((value) => {
-      scrollNode.scrollTo({ x: 0, y: value.y, animated: false });
+      scrollNode?.scrollTo({ x: 0, y: value.y, animated: false });
     });
-
-    if (y < -20 && !constants.isAndroid && !refreshControl) this.spring(y);
+    //@TODO Check that
+    if (y < -20 && !constants.isAndroid && !refreshControl) this.spring();
+    // if (y < -20 && !constants.isAndroid && !refreshControl) this.spring(y);
```

Comment:
Are you able to remove "TODO" and comments?

Reply:
Done